client/knowledge_base_search/src/__test__/containers/ResultContainer/ResultContainer.test.jsx

```diff
@@ -8,12 +8,40 @@ describe('<ResultContainer />', () => {
     resultText: 'result',
     resultRank: 1,
   };
+  let wrapper;
 
   it('renders without crashing', () => {
     const div = document.createElement('div');
     ReactDOM.render(<ResultContainer {...props} />, div);
   });
 
+  describe('when the result is higher than the originalRank', () => {
+    const propsWithGreaterResultRank = Object.assign({}, props, {
+      resultRank: 1,
+      originalRank: 0,
+    });
+
+    it('says that Watson moved the rank up', () => {
+      wrapper = shallow(<ResultContainer {...propsWithGreaterResultRank} />);
+      const rankDisplay = wrapper.find('.training_comparison--rank-up');
+      expect(rankDisplay).toHaveLength(1);
+    });
+  });
+
+
+  describe('when the originalRank is higher than the resultRank', () => {
```

Comment:
what happens when the `originalRank === resultRank` ?

Reply:
nothing, per design's instructions

Reply:
im implying that you should have a test showing that